app/models/project/Project.scala

```diff
@@ -103,6 +104,21 @@ object ProjectSQLDAO extends SQLDAO[ProjectSQL, ProjectsRow, Projects] {
       parsed <- parse(r)
     } yield parsed
 
+  def findUsersWithOpenTasks(name: String)(implicit ctx: DBAccessContext): Fox[List[String]] =
+    for {
+      accessQuery <- readAccessQuery
+      rSeq <- run(sql"""select u.email
```

Comment:
Why do you return just the user email and not the whole user information?

Reply:
there is no frontend for this route (yet), this list of emails was created directly for heiko’s request